rpcs3/Emu/CMakeLists.txt

```diff
@@ -80,12 +81,12 @@ target_link_libraries(rpcs3_emu
         3rdparty::pugixml)
 
 if(MSVC)
-    set_source_files_properties("../../Utilities/JIT.cpp" PROPERTIES
+    set_source_files_properties("../../Utilities/JITLLVM.cpp" PROPERTIES
```

Comment:
shouldn't this also be done for the JITASM.cpp ?

Reply:
I don't know. I thought it was necessary because we compile LLVM without RTTI, but I'm not sure.

Reply:
Anyway I did it for `JITASM.cpp` too.